R/snapshot-parameter.R

```diff
@@ -56,8 +56,8 @@ SnapshotParameter <- R6::R6Class(
   )
 )
 
-createSnapshotParameter <- function(value, unit) {
-  if (is.null(value)) {
+.createSnapshotParameter <- function(value, unit) {
```

Comment:
@msevestre This is what you proposed here, right?
https://github.com/esqLABS/esqlabsR/pull/183

Reply:
Definitely a better solution than converting all nas to NULLs by hand.

Reply:
yes